channels_test.go

```diff
@@ -172,7 +172,7 @@ func TestGetChannelInformation(t *testing.T) {
 		c := newMockClient(testCase.options, newMockHandler(testCase.statusCode, testCase.respBody, nil))
 
 		resp, err := c.GetChannelInformation(&GetChannelInformationParams{
-			BroadcasterID: testCase.BroadcasterID,
+			BroadcasterIDs: []string{testCase.BroadcasterID},
```

Comment:
Since we're keeping both struct fields, we need to have separate tests for both use cases.

Please revert this change and instead add a new test case to cater to `BroadcasterIDs`.